static/app/components/demoSandboxButton.tsx

```diff
@@ -0,0 +1,94 @@
+import Button from 'sentry/components/button';
+import {Organization} from 'sentry/types';
+import trackAdvancedAnalyticsEvent from 'sentry/utils/analytics/trackAdvancedAnalyticsEvent';
+import withOrganization from 'sentry/utils/withOrganization';
+
+type Props = {
+  /**
+   * The deep link scenario
+   */
+  scenario:
+    | 'performance'
+    | 'releases'
+    | 'alerts'
+    | 'discover'
+    | 'dashboards'
+    | 'projects'
+    | 'oneDiscoverQuery'
+    | 'oneIssue'
+    | 'oneBreadcrumb'
+    | 'oneStackTrace'
+    | 'oneTransaction'
+    | 'oneWebVitals'
+    | 'oneTransactionSummary'
+    | 'oneRelease';
+  organization: Organization;
+  buttonText: string;
```

Comment:
With my suggestion below we wouldn't need this and you would just pass children into the button.